src/test/HostActivation.Tests/NativeHosting/ComponentActivation.cs

```diff
@@ -39,11 +39,9 @@ public void CallDelegate(bool validPath, bool validType, bool validMethod)
                 validMethod ? sharedState.ComponentEntryPoint1 : "BadMethod",
             };
             CommandResult result = Command.Create(sharedState.NativeHostPath, args)
-                .CaptureStdErr()
-                .CaptureStdOut()
-                .EnvironmentVariable("COREHOST_TRACE", "1")
-                .EnvironmentVariable("DOTNET_ROOT", sharedState.DotNetRoot)
-                .EnvironmentVariable("DOTNET_ROOT(x86)", sharedState.DotNetRoot)
+                .EnableTracingAndCaptureOutputs()
```

Comment:
This would be worth adding a helper onto the `sharedState` to execute the native host with specified args - or create a command for that - basically everything up to the `.Execute`. Seems that most tests in this class repeat the exact same thing. Would make this change much more reasonable.

Reply:
+1.  I had written the same suggestion but then saw this comment.